src/Hal/Metric/Information/AbstractInformation.php

```diff
@@ -0,0 +1,128 @@
+<?php
```

Comment:
This class looks really complex, and has many responsabilitites

+ each class is a value object and represents a cluster of metrics
+ but each class ahs also the responsability of rendering itself as string and HTML

I think that the methods `*Html()` should be moved in a renderer or directly used in the HTML templates